subprojects/functional/src/main/java/org/gradle/internal/Try.java

```diff
@@ -234,19 +248,22 @@ public T getOrMapFailure(Function<Throwable, T> f) {
             return Optional.of(failure);
         }
 
+        @SuppressWarnings("unchecked")
```

Comment:
I would constrain these warnings to the `return` statements.

Reply:
The reason I'm using annotations is because they are visible to the compiler, too. IntelliJ's comments don't work like that.

Reply:
I see. Kotlin has that capability, I assume Java had it too.